README.md

```diff
@@ -98,6 +99,13 @@ one or more of:
     `stderr`/`stdout` matches ignore leading/trailing whitespace and newlines,
     but are case sensitive.
 
+Test commands can alter the general command by specifying zero or more of the
+following:
+
+  * `extra-args: <arg 1> [... <arg n>]`, where each space separated argument
+    will be passed to the command in question in addition to the command lines
+    specified as part of `test_cmds`.
+
```

Comment:
This might be obvious, but it may be worth mentioning that `extra-args` come *after* the arguments specified in the test runner.

Reply:
Good point.

Reply:
See if https://github.com/softdevteam/lang_tester/pull/27/commits/74b539abfa55323afd47e1927f2e1e8fc0be3b43 works for you.

Reply:
One slight nit: I would say "... as part of the `test_cmds` function", because it was a while since it was mentioned and people might have forgot what it is this far down the README!

Reply:
Fixed in https://github.com/softdevteam/lang_tester/pull/27/commits/c31c8b8dd7313b465ec42ac843da1698ccce4ba2.